prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -352,7 +370,7 @@ class FHIRRouter(
                 report,
                 receiver,
                 ReportStreamFilterType.PROCESSING_MODE_FILTER,
-                false
+                defaultResponse = true
```

Comment:
Just wanted to verify the rationale for this change. Are we sure it doesn't bring unexpected changes?